### ISSUE TITLE
More specific description based on genders

### ISSUE THREAD

**Issue by leyan** [bartfeenstra/betty#901]
With the few small issues I solved, my website is starting to look good! However, one large issue remains, as I am aiming for a French version. The gender neutral description really does not work and people will not understand a sentence like "Ce sont les enfants de ..." ("They are the child of ...") or "Ils ont grandi avec un frère ou une sœur." ("They grew up with a sibling."). 

The neutral "they" does not exist in French, and there is no word for gender neutral sibling in French ("un frère ou une sœur" sounds like we do not know, which is clearly not true as the information is on the next column).

I had a look to change that, but it seems like you do not even load the gender information from Gramps, is that correct? It is not used anywhere? Would you be against loading it, so that we can know this information and adapt the string to be displayed when the gender is known?

**Comment by bartfeenstra** [comment]
Thanks for raising this issue :heart: You are right in observing that no sex or gender information is currently loaded by Betty. The simple reason is that I personally did not care for it, and that sex and gender are complex social topics, requiring more than just *male* or *female*. We would need a pluggable sex/gender type system that allows extensions as well as individual site owners to define additional types of sexes and genders if that is necessary for their family trees. Additionally, we would need a way to map Gramps sexes/genders (plain Gramps sexes are binary, so we'd need to use tags for this, just like we do for privacy) to Betty sexes/genders. Furthermore, we need to remember that someone's *registered* sex/gender can change over time, and that any solution we implement acknowledges this.

None of this is to say that we should not or must not implement this. **I would love for us to support this**, but we need the kind of inclusivity that allows for more sexes and genders than just two. I would be very happy to review and test any proposal that addresses these concerns.

**Comment by bartfeenstra** [comment]
This also means that predefined strings are more difficult to handle than you may expect: if the sexes/genders in a family tree cannot be known beforehand, we cannot reliably provide the translatable strings for each and every one of them.

@patlx contributed the original French translations. Patrice and @leyan, my knowledge of French is limited. Can you, in this issue, explain if there are indeed any options to write French without explicitly specifying gender, and preferably without sounding odd? I am fully aware that this may be difficult, as it would be in my own native language.

**Comment by bartfeenstra** [comment]
Hi @patlx and @leyan ! Do you have any insights to share? 😊

**Comment by leyan** [comment]
There is no natural way to express "I don't think the gender of the person is important" in French. We should include this information if it is available. However, to really avoid using a gendered pronoun, you have two ways: 
* Writing  "Il ou elle", like "He or she". It can work, but it really reads like the author does not know.
* Using "Iel", which is a neologism still quite limited to militant writings (LGBT+ circles, left-wing politics, etc.) and not (yet?) entered in general usage. See [the wikipedia page](https://en.wikipedia.org/wiki/Iel_(pronoun))

To read naturally to a French speaker, I think we should get and use the information when available, use "Il ou elle" when not known, and reserve "iel" or something similar when the gender is explicitly defined as something else (when Gramps will be able to support it, see https://gramps-project.org/bugs/view.php?id=5730).

**Comment by bartfeenstra** [comment]
I'd like to remind everyone that Betty is a safe space and that we are welcoming and inclusive to humans in all their diversity. I will happily discuss technical and linguistic solutions that help us express sex and gender. However, words like _militant_ have no place in this community.

**Comment by leyan** [comment]
Is it a false friend ? This is not offensive in french. Maybe I should have used activists ? Anyway, do as you wish.

**Comment by patlx** [comment]
Hi,

Yes, "il ou elle" is probably the good way to use in reports.

Regards,

Patrice

**Comment by Jellby** [comment]
How about "this person" (or equivalent in other languages)? Or "this individual", if Betty/Gramps is also used for non-human trees (but then maybe "child" would also need to be changed).

**Comment by bartfeenstra** [comment]
> How about "this person" (or equivalent in other languages)? Or "this individual", if Betty/Gramps is also used for non-human trees (but then maybe "child" would also need to be changed).

Hello, and welcome! The problem is not with English, which has a fitting way to describe a single person without referring to their gender (singular *they*), but instead lies with those languages that do not (yet) provide a (commonly used) way to refer to people without using explicit gender markers, such as gendered personal pronouns, suffixes, etc.

Any solution would be additional to the current situation, because if even and when a language (and its Betty translations) can express all of our translatable strings using gendered personal pronouns (which would greatly increase the amount of translation work that is to be done, **for each language**), we must still support the case where we must refer to a person without knowing anything about their gender.

In short, any gendering we add to Betty in the future is an addition, and cannot replace the gender-neutral approach we already have.

**Comment by bartfeenstra** [comment]
I opened https://github.com/bartfeenstra/betty/issues/901, which adds gender support, a handful of built-in genders, and the ability to alter those and/or add your own genders on a per-project basis. In future pull requests, as we improve the Cotton Candy theme, we can start displaying those on people's site's by default. Until then, the underlying data will be accessible in all templates so you can add your own project overrides.

With regards to translatable strings, considering the French translation is currently [less than half complete](https://betty.readthedocs.io/0.4.x/usage/translation/), let's not add variations of these that will add to the workload. However, in your project templates you are free to add whatever text of your own you like.
